packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -94,9 +85,8 @@ const patternSource = computed(() => {
             :maxlength="props.maxlength"
             :aria-label="props.hideLabel ? props.label : undefined"
             :title="props.hideLabel ? props.label : undefined"
-            @change="handleChange"
-            @focus="emit('focus')"
-            @blur="emit('blur')"
+            @focus="emit('inputFocus')"
+            @blur="emit('inputBlur')"
```

Comment:
Would prefer to still call them focus and blur and pass the native event with them

Reply:
But thats my personal opinion, maybe @JoCa96 disagrees on that

Reply:
The issue is that `focus` and `blur` do not bubble, so currently the user cannot listen for them on the `OnyxInput` component.
1. We don't provide any emits for `focus` and `blur`
**Reasoning:** While these don't bubble, they still have a capture phase, so `@focus.capture=...` and `@blur.capture=...` will still work. Also, `focusin` and `focusout` do bubble and could be used instead.
**Caveat:** Only works well for `OnyxInput`, where we have a single focusable element in our component.
2. We use `const { rootAttrs, restAttrs } = useRootAttrs();` and bind `restAttrs` to the input element.
**Reasoning:** This way all event-listeners and attributes of the user are bound to the input element. Also works for all other event-listeners.
**Caveat:** Might infer with our own aria-bindings 
3. Rename the emits with a common name for all our components: I would propose `activate` and `deactivate`. These event names are not colliding with any native event 
namings, which is the main purpose of the renaming.

I would personally prefer to do option 1 for now, create a follow-up ticket for 3 and wait for user-feedback.

Reply:
I removed the focus and blur events, but I haven't created a follow up ticket for the third point.